From: Danny Kukawka <danny.kukawka@web.de>
Date: Wed, 23 May 2007 18:05:17 +0000 (+0200)
Subject: updated allowed libparted versions
X-Git-Url: http://gitweb.freedesktop.org/?p=hal.git;a=commitdiff;h=366ab0006278f6c23fc3c33dea7eb61a42fe941e

updated allowed libparted versions

Updated again the list of allowed liparted version by add v1.8.7
---

Index: b/configure.in
===================================================================
--- a/configure.in	2007-04-30 14:58:50.000000000 -0400
+++ b/configure.in	2007-08-01 17:38:20.000000000 -0400
@@ -193,7 +193,7 @@
    AC_DEFINE(USE_PARTED,1,[Whether libparted is to be used])
    AC_CHECK_LIB(uuid, uuid_generate, [], AC_MSG_ERROR([*** uuid library (libuuid) not found]))
    AC_CHECK_LIB(dl, dlopen,          [], AC_MSG_ERROR([*** dl library (libdl) not found]))
-   AC_MSG_CHECKING([for libparted == 1.7.1, 1.8.0, 1.8.1, 1.8.2, 1.8.6])
+   AC_MSG_CHECKING([for libparted == 1.7.1, 1.8.0, 1.8.1, 1.8.2, 1.8.6, 1.8.7])
    LDFLAGS=-lparted
    AC_TRY_RUN(
    #include <stdio.h>
@@ -211,12 +211,13 @@
 	    (major == 1 && minor == 8 && micro == 0) ||
 	    (major == 1 && minor == 8 && micro == 1) ||
 	    (major == 1 && minor == 8 && micro == 2) ||
-	    (major == 1 && minor == 8 && micro == 6))
+	    (major == 1 && minor == 8 && micro == 6) ||
+	    (major == 1 && minor == 8 && micro == 7))
 	   return 0;
 
 	return 1;
     }
-	,AC_MSG_RESULT( ),AC_MSG_ERROR([*** Requires libparted == 1.7.1/1.8.0/1.8.1/1.8.2 or 1.8.6]) )
+	,AC_MSG_RESULT( ),AC_MSG_ERROR([*** Requires libparted == 1.7.1/1.8.0/1.8.1/1.8.2/1.8.6 or 1.8.7]) )
     PARTED_LIBS=-lparted
     AC_SUBST(PARTED_LIBS)
     LDFLAGS=
